apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/shard/observer/ShardObserverIntegrationTest.java

```diff
@@ -46,7 +46,7 @@
     @Test
     void testDoShardByAsyncEvent() {
         Mockito.doReturn(heightConfig).when(blockchainConfig).getCurrentConfig();
-        Mockito.doReturn(4072*1024*1024L).when(mock(Runtime.class)).totalMemory(); // give it more then 3 GB
+        //Mockito.doReturn(4072*1024*1024L).when(mock(Runtime.class)).totalMemory(); // give it more then 3 GB
```

Comment:
We cant mock static methods, need to completely remove it

Reply:
with mock-maker-inline, `Mockito.doReturn(4072*1024*1024L).when(mock(Runtime.class)).totalMemory()` throws an exception

Reply:
done